app/config/services.yml

```diff
@@ -24,6 +24,11 @@ services:
         tags:
             - { name: twig.extension }
 
+    app.twig.extension_intl:
+        class: Twig_Extensions_Extension_Intl
```

Comment:
you should make the service private so that it gets inline when optimizing the container